index.js

```diff
@@ -32,6 +32,13 @@ module.exports = function (buf) {
 		};
 	}
 
+	if (((buf[0] === 0x49 && buf[1] === 0x49 && buf[2] === 0x2A && buf[3] === 0x0) || (buf[0] === 0x4D && buf[1] === 0x4D && buf[2] === 0x0 && buf[3] === 0x2A)) && buf[8] === 0x43 && buf[9] === 0x52) {
```

Comment:
Is there a reason you added this here and not at the bottom? If there is let's add a comment about that.